src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs

```diff
@@ -770,6 +785,12 @@ await using (var stream = GetInputStream(decompressionStream, snapshotEncryption
 
                     // need to enable revisions before import
                     database.DocumentsStorage.RevisionsStorage.InitializeFromDatabaseRecord(smugglerDatabaseRecord);
+                },
+                onSubscriptionAction: subscriptionState =>
```

Comment:
this is used in other places to substitute the original code, let's do the same here, so pass the actions and convert to func